src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -140,9 +140,7 @@ const WorkspaceContainer = ({
       setSharingWorkspace, setShowLockWorkspaceModal
     }),
     div({ role: 'main', style: Style.elements.pageContentContainer },
-
-      // TODO: When we switch this over to all tabs, ensure other workspace tabs look the same when inside these divs
-      (isAnalysisTabVisible() && _.includes(activeTab, contextBarTabs) ?
+      (isAnalysisTabVisible() ?
         [div({ style: { flex: 1, display: 'flex' } }, [
           div({ style: { flex: 1, display: 'flex', flexDirection: 'column' } }, [
             children
```

Comment:
@jdcanas could you remove `setDeletingWorkspace, setCloningWorkspace, setSharingWorkspace, and setShowLockWorkspaceModal` from being passed to `ContextBar` below at line 149? `ContextBar` does not use them… I think it crept in through refactoring of the workspace menu.

Reply:
Thanks!